platform/ios/src/MGLMapView.mm

```diff
@@ -4383,7 +4383,7 @@ - (void)updateAnnotationViews
                 }
                 
                 CGPoint center = [self convertCoordinate:annotationContext.annotation.coordinate toPointToView:self];
-                [annotationView setCenter:center pitch:self.camera.pitch];
+                [annotationView setCenter:center pitch:self.camera.pitch maxPitch:MGLDegreesFromRadians(mbgl::util::PITCH_MAX)];
```

Comment:
`mbgl::util::PITCH_MAX` is constant, so instead of passing it into this method, use it directly in `-setCenter:pitch:`. You’ll need to rename MGLAnnotationView.m to have a .mm extension so you can use Objective-C++ namespaces.